### PR TITLE
add top level readme

### PR DESCRIPTION
**Pull request by acharb**
[ticket](https://stellarorg.atlassian.net/browse/WAL-1483?atlOrigin=eyJpIjoiNDY3NTNiMmZiYjU0NDdhNWJhZDk5NWExYjczZmJkZGYiLCJwIjoiaiJ9)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+applications on the Stellar network faster. To jump right into how the main
+typescript-wallet-sdk module works you can
+[go here](./@stellar/typescript-wallet-sdk/README.md).
+
+## Yarn Workspaces
+
+This repo uses a yarn worspace to create a monorepo that includes the main
+wallet sdk, and two modules for extending functionality:
+
+- [typescript-wallet-sdk](./@stellar/typescript-wallet-sdk/README.md) - the main
```
Comment (Ifropc) [inline README.md:13]: Probably a good idea to add yarn install command for all listed sub projects

**Review by Ifropc — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (acharb) [reply README.md:13]: good call
https://github.com/stellar/typescript-wallet-sdk/pull/136/commits/bbd2f6cc4fbb7635a2e52e268e7229eeae46b5a7

**Comment by acharb** [comment]
test is failing bc of error in testanchor, I [pinged](https://stellarfoundation.slack.com/archives/C036T11AY84/p1715738998236649) anchor-eng team about

**Review by CassioMG — APPROVED** [review]
LGTM

Should we mention the sep24 example we have for the main package?

**Pull request merged**